admin-frontend/open_admin_app/lib/widgets/user/list/list_users_bloc.dart

```diff
@@ -61,6 +61,14 @@ class ListUsersBloc implements Bloc {
         .deletePerson(personId, includeGroups: includeGroups);
   }
 
+  Future<String?> resetApiKey(Person person) async {
+    return mrClient.personServiceApi
+        .resetSecurityToken(person.id!.id)
+        .then((response) {
+      return response.token;
+    }).catchError((e, s) {});
```

Comment:
this is incorrect code - it should return null on a fail, you are only returning a value on a success, i'm not sure how this code compiles!